php/templates/hpa.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.hpa.enabled -}}
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "php.fullname" . }}
+  labels: {{ include "php.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    apiVersion: apps/v1
+    name: {{ template "php.fullname" . }}
+  minReplicas: {{ .Values.hpa.minReplicas }}
+  maxReplicas: {{ .Values.hpa.maxReplicas }}
+  targetCPUUtilizationPercentage: {{ .Values.hpa.targetCPUUtilizationPercentage }}
```

Comment:
https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/controller-hpa.yaml
There is no problem with `cpu`, but `memoryUtilization` may be acceptable in terms of versatility.

And we can use `autoscaling/v2beta1` in 1.14
https://v1-14.docs.kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/

Reply:
@cw-sakamoto 
I fixed it with [examples](https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/controller-hpa.yaml).